test/spdy/parser-test.js

```diff
@@ -0,0 +1,445 @@
+var assert = require('assert');
+
+var transport = require('../../');
+var spdy = transport.protocol.spdy;
+
+describe('SPDY Parser', function() {
+  var parser;
+
+  beforeEach(function() {
+    var pool = spdy.compressionPool.create();
+    parser = spdy.parser.create({});
+    var comp = pool.get('3.1');
+    parser.setCompression(comp);
```

Comment:
I think it might be a good idea to skip preface here instead of duplicating the logic in `pass`/`fail`.